webapp-django/crashstats/signature/templates/signature/signature_summary.html

```diff
@@ -77,6 +77,9 @@
                 {% endfor %}
             </tbody>
         </table>
+        <em>
```

Comment:
`em` is not a blocking tag. By the way, `em` (and `strong`) is a relic from the XHTML days. The more modern one to use is `i` (and `b`).